libcodechecker/orm_model.py

```diff
@@ -191,6 +189,11 @@ class Report(Base):
 
     # TODO: multiple messages to multiple source locations?
     checker_message = Column(String)
+    detection_status = Column(String)
```

Comment:
In this "WIP" version I had problems in Alembic with enum types and I didn't want to struggle with it. Fortunately in version6 no DB migration will be used at this point, so this will be fixed soon.